SYCL/lit.cfg.py

```diff
@@ -176,37 +176,52 @@
     config.sycl_be = config.sycl_be[3:]
 config.sycl_be = config.sycl_be.lower()
 
+# Replace deprecated backend names
+deprecated_names_mapping = {'cuda'       : 'ext_oneapi_cuda',
+                            'hip'        : 'ext_oneapi_hip',
+                            'level_zero' : 'ext_oneapi_level_zero',
+                            'esimd_cpu'  : 'ext_intel_esimd_emulator'}
```

Comment:
I wonder BTW, why would we keep the deprecated esimd_cpu name rather than simply drop it?

Reply:
we support all deprecated BE until they are finally removed.